src/components/MarketList/index.js

```diff
@@ -243,28 +248,27 @@ class MarketList extends Component {
             </div>
           </div>
         </div>
-        {process.env.WHITELIST[defaultAccount] && (
-          <div className="marketListPage__controls">
-            <div className="container">
-              <div className="row">
-                <div className="col-xs-10 col-xs-offset-1 col-sm-12 col-sm-offset-0">
-                  <InteractionButton
-                    onClick={this.handleCreateMarket}
-                    className="marketStats__control btn btn-default"
-                    whitelistRequired
-                  >
-                    Create Market
-                  </InteractionButton>
-                </div>
+        <div className="marketListPage__controls">
+          <div className="container">
+            <div className="row">
+              <div className="col-xs-10 col-xs-offset-1 col-sm-12 col-sm-offset-0">
+                <InteractionButton
+                  onClick={this.handleCreateMarket}
+                  className="marketStats__control btn btn-default"
+                  whitelistRequired
+                >
+                  Create Market
+                </InteractionButton>
               </div>
             </div>
           </div>
-        )}
-        <div className="marketListPage__markets">
-          <div className="container">
-            <div className="row">
-              {this.renderMarkets()}
-              {this.renderMarketFilter()}
+          )}
+          <div className="marketListPage__markets">
+            <div className="container">
+              <div className="row">
+                {this.renderMarkets()}
```

Comment:
I do not know if you are refactoring now but those functions (```this.renderMarkets() and this.renderMarketFilter()```) should be private components.

Reply:
@apanizo We will do it after the backmerge is complete

Reply:
Sure, no problem.